docs/docs/spec.md

```diff
@@ -81,6 +24,52 @@ This document also links to other documents. Those documents are non-normative,
 except for when we link to the Java Language Specification to defer to its
 rules.
 
+> ### The word "nullable"
+>
+> In this doc, we aim not to refer to whether a type "is nullable." Instead, we
+> draw some distinctions, creating at least four kinds of "Is it nullable?"
+> questions we can ask for any given type usage:
+>
+> 1.  What is the [augmented type] of that type usage?
```

Comment:
I'm still not convinced that this is an "is it nullable" question. This is how JSpecify proposes that people _answer_ any of the "is it nullable" questions. But okay.

Reply:
Yeah, I think we've made a couple attempts at how to phrase this. I'm sure we could find something that works better.